source/math/FloatSize3D.ooc

```diff
@@ -46,12 +46,12 @@ FloatSize3D: cover {
 		)
 	}
 	angle: func (other: This) -> Float {
-		(this scalarProduct(other) / (this norm * other norm)) clamp(-1.0f, 1.0f) acos() * (this width * other height - this height * other width < 0.0f ? -1.0f : 1.0f)
+		(this scalarProduct(other) / (this norm * other norm)) acos() * (this width * other height - this height * other width < 0.0f ? -1.0f : 1.0f)
```

Comment:
This really "should" be fine, but to be absolutely sure we don't get something weird due to floating point arithmetic as @sebastianbaginski points (no pun intended) out, I will keep `clamp` here.